README.rst

```diff
@@ -7,13 +7,14 @@ NuoDB Client Package
 
 .. contents::
 
-The NuoDB Client Package bundles the latest publicly-available NuoDB_ database
-client versions into a single downloadable file. The official versions of the
-NuoDB Client Package file are available from the NuoDB `GitHub Releases`_
+The NuoDB Client Package bundles the latest publicly available NuoDB_ database
+client versions into a single downloadable file. Separate packages for each
+components are also available. The official versions of the NuoDB Client Package
+files are available from the NuoDB `GitHub Releases`_
 page.
 
-You can also use this project to create a package containing customized sets
-of NuoDB Client packages.
+You can also use this project to create packages containing customized sets
+of NuoDB Client individual packages.
```

Comment:
Did you mean "individual NuoDB Client packages"?

Reply:
That is the idea. Not sure if this sentence is doing a good job explaining it. Feel free to rephrase it if necessary.